modules/system/console/OctoberInstall.php

```diff
@@ -130,6 +131,23 @@ protected function setupAdvancedValues()
         $this->writeToConfig('app', ['debug' => $debug]);
     }
 
+    protected function askToInstallPlugins() {
+        if ($this->confirm('Do you wish to install October.Drivers plugin?', false)) {
```

Comment:
That would bring the language more in line with the other confirmations requested in the command